ledger-api/grpc-definitions/com/daml/ledger/api/v1/events_service.proto

```diff
@@ -0,0 +1,364 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+
+/*
+Satisfied Design goals:
+- can co-exist with existing API services for single-domain participants
+- can be served with the 1.17 IndexDB
```

Comment:
I don't think this should be a design goal for a major version of the ledger API. With v2, we should get to the point of explicit information (https://docs.google.com/document/d/1c1svk_JD1-KTjNjla-xVduJKQJAb37qnsyTzr-7_dek/edit#heading=h.yowjto3ekggr)